ui/shared/components/graph/IGV.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import igv from 'igv'
 import styled from 'styled-components'
-import style from 'igv/igv.css' //eslint-disable-line no-unused-vars
```

Comment:
why is this no longer neccessary?

Reply:
IGV no longer has a css dependency (https://github.com/igvteam/igv.js?files=1#installation)